Classes/Controller/DocumentsController.php

```diff
@@ -78,12 +92,13 @@ public function indexAction(string $workspaceName = 'live'): void
         // TODO Check that public access is only granted to live workspace (or content api is completely restricted by API key)
 
         $documents = [];
+        $filter = $this->settings['documentsFilter'] ?? [];
 
         $site = $this->getActiveSite();
         $siteNodeName = $site->getNodeName();
         foreach ($this->nodeEnumerator->siteNodeInContexts($site, $workspaceName) as $siteNode) {
             foreach ($this->nodeEnumerator->recurseDocumentChildNodes($siteNode) as $documentNode) {
-                if ($documentNode->getNodeType()->isOfType('Neos.Neos:Shortcut')) {
+                if (in_array($documentNode->getNodeType()->getName(), $filter)) {
```

Comment:
We should keep using `$documentNode->getNodeType()->isOfType($nodeTypeName)` and iterate all node type names, since that will include super types when matching the type (which is what you would expect in Neos in other situations).

Reply:
Done. I also now throw an exception when the ignoredNodetypes is not an array.